saleor/extensions/base_plugin.py

```diff
@@ -206,6 +207,11 @@ def _update_config_items(
             for config_item_to_update in configuration_to_update:
                 if config_item["name"] == config_item_to_update.get("name"):
                     new_value = config_item_to_update.get("value")
+                    item_type = config_item.get("type")
```

Comment:
What do you think about a separate function like - `clean_configuration` which should be run before the update of configuration items? This would be helpful for future plugins.